tools/scripts/dpm.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import sys
+
+# sign governance and permissions
+
+dpm_path = './DPM'
+
+class CertificateAuthority:
+    def __init__(self, name):
+        self.name = name
+        self.path = f'{dpm_path}/ca/{name}'
```

Comment:
If you turn this into a [`Path`](https://docs.python.org/3/library/pathlib.html), the rest of the code could cleaner and more portable.